Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
 # Install general dependencies
-RUN apt-get update && apt-get install -y build-essential git curl libssl-dev libreadline-dev zlib1g-dev
+RUN apt-get update && apt-get install -y build-essential git curl libssl-dev libreadline-dev zlib1g-dev python3-dev
```

Comment:
The added python3 dep is the only thing I don't get. Everything else makes sense to me.